### ISSUE TITLE
8.1.2344 Build failure on macOS 10.14.6

### ISSUE THREAD

**Issue by chdiza** [vim/vim#5270]
On macOS 10.14.6, trying to build 8.1.2446 bombs out quickly.  Errors pasted in below.  Bisection reveals that the culprit is: 8.1.2344, i.e., 6a228c64639.

```
gcc -c -I. -Iproto -DHAVE_CONFIG_H   -DMACOS_X -DMACOS_X_DARWIN  -g -O2 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1        -o objects/os_macosx.o os_macosx.m
In file included from os_macosx.m:26:
In file included from ./vim.h:237:
In file included from ./os_unix.h:56:
In file included from /Users/chdiza/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/libc.h:44:
In file included from /Users/chdiza/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/sys/mount.h:76:
In file included from /Users/chdiza/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/sys/attr.h:42:
/Users/chdiza/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/sys/ucred.h:101:2: error:
      unknown type name 'u_int'
        u_int   cr_version;             /* structure layout version */
        ^
In file included from os_macosx.m:26:
In file included from ./vim.h:237:
In file included from ./os_unix.h:56:
In file included from /Users/chdiza/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/libc.h:44:
In file included from /Users/chdiza/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/sys/mount.h:76:
/Users/chdiza/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/sys/attr.h:77:2: error:
      unknown type name 'u_short'; did you mean 'short'?
        u_short bitmapcount;                    /* number of attr. bit s...
        ^
/Users/chdiza/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/sys/attr.h:558:2: error:
      unknown type name 'u_long'; did you mean 'long'?
        u_long                          maxmatches;
        ^
/Users/chdiza/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/sys/attr.h:571:2: error:
      unknown type name 'u_char'; did you mean 'char'?
        u_char                          ss_fsstate[548];        // fs private
        ^
gcc -c -I. -Iproto -DHAVE_CONFIG_H   -DMACOS_X -DMACOS_X_DARWIN  -g -O2 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1        -o objects/os_mac_conv.o os_mac_conv.c
gcc -c -I. -Iproto -DHAVE_CONFIG_H   -DMACOS_X -DMACOS_X_DARWIN  -g -O2 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1        -o objects/channel.o channel.c
In file included from os_mac_conv.c:18:
In file included from ./vim.h:237:
In file included from ./os_unix.h:56:
In file included from /Users/chdiza/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/libc.h:44:
In file included from /Users/chdiza/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/sys/mount.h:76:
In file included from /Users/chdiza/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/sys/attr.h:42:
/Users/chdiza/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/sys/ucred.h:101:2: error:
      unknown type name 'u_int'
        u_int   cr_version;             /* structure layout version */
        ^
In file included from os_mac_conv.c:18:
In file included from ./vim.h:237:
In file included from ./os_unix.h:56:
In file included from /Users/chdiza/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/libc.h:44:
In file included from /Users/chdiza/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/sys/mount.h:76:
/Users/chdiza/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/sys/attr.h:77:2: error:
      unknown type name 'u_short'; did you mean 'short'?
        u_short bitmapcount;                    /* number of attr. bit s...
        ^
/Users/chdiza/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/sys/attr.h:558:2: error:
      unknown type name 'u_long'; did you mean 'long'?
        u_long                          maxmatches;
        ^
/Users/chdiza/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/sys/attr.h:571:2: error:
      unknown type name 'u_char'; did you mean 'char'?
        u_char                          ss_fsstate[548];        // fs private
        ^
In file included from channel.c:13:
In file included from ./vim.h:237:
In file included from ./os_unix.h:56:
In file included from /Users/chdiza/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/libc.h:44:
In file included from /Users/chdiza/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/sys/mount.h:76:
In file included from /Users/chdiza/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/sys/attr.h:42:
/Users/chdiza/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/sys/ucred.h:101:2: error:
      unknown type name 'u_int'
        u_int   cr_version;             /* structure layout version */
        ^
In file included from channel.c:13:
In file included from ./vim.h:237:
In file included from ./os_unix.h:56:
In file included from /Users/chdiza/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/libc.h:44:
In file included from /Users/chdiza/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/sys/mount.h:76:
/Users/chdiza/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/sys/attr.h:77:2: error:
      unknown type name 'u_short'; did you mean 'short'?
        u_short bitmapcount;                    /* number of attr. bit s...
        ^
/Users/chdiza/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/sys/attr.h:558:2: error:
      unknown type name 'u_long'; did you mean 'long'?
        u_long                          maxmatches;
        ^
/Users/chdiza/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/sys/attr.h:571:2: error:
      unknown type name 'u_char'; did you mean 'char'?
        u_char                          ss_fsstate[548];        // fs private
        ^
4 errors generated.
make[1]: *** [objects/channel.o] Error 1
make[1]: *** Waiting for unfinished jobs....
4 errors generated.
make[1]: *** [objects/os_mac_conv.o] Error 1
4 errors generated.
make[1]: *** [objects/os_macosx.o] Error 1
make: *** [first] Error 2
```

**Comment by chrisbra** [comment]
I assume this is fixed by 84f903326d44db9b75fc3a39d4866f636f9ad4cd?

**Comment by chdiza** [comment]
Yes, apparently that patch was pushed while I was in the middle of bisection and I didn't realize it.
